ChangeLog

```diff
@@ -33,6 +33,7 @@
  * Deprecated classes UserDefinedPair, HistogramPair
  * Removed SensitivityAnalysis class
  * Deprecated SLSQP, LBFGS and NelderMead classes in favor of NLopt class
+ * Deprecated LAR in favor of LARS
 
```

Comment:
@sofianehaddad, it's renamed in the c++, but I added LAR(LARS) in lars.i, I should have mentioned that